apps/nextjs-website/src/lib/homepage.ts

```diff
@@ -40,12 +55,27 @@ type StaticHomepage = typeof translations.homepage;
 
 export const makeHomepageProps = (
   strapiHomepage: StrapiHomepage,
+  strapiEndpoint: string,
   staticHeader: StaticHeader,
   staticHomepage: StaticHomepage
 ): HomepageProps => ({
   ...makeHomepagePropsFromStatic(staticHeader, staticHomepage),
   comingsoonDocumentation:
     strapiHomepage.data.attributes.comingsoonDocumentation,
+  productsShowcase: {
+    title: strapiHomepage.data.attributes.productsShowcase.title,
+    products: strapiHomepage.data.attributes.productsShowcase.products.data.map(
+      (product) => ({
+        name: product.attributes.name,
+        description: product.attributes.description,
+        slug: product.attributes.slug,
+        logo: {
+          ...product.attributes.logo.data.attributes,
+          url: `${strapiEndpoint}${product.attributes.logo.data.attributes.url}`,
```

Comment:
This solution may not work in every situation since the media will be served behind a CDN. To move forward, we can either create an activity or modify the implementation and assume that the full path will be returned (in this case it will not work locally). In the first case, it's a good idea to include the PR link in the task, and vice versa.

Reply:
I will adopt the solution without `strapiEndpoint` that doesn't work on localhost, but only in dev and possibly in production environments